sote/game/scenes/game/tile-inspector.lua

```diff
@@ -312,7 +297,7 @@ local function realm_widget(gam, tile, panel)
 end
 
 local function main_panel(gam, tile, panel)
-	ui.panel(panel)
+	-- ui.panel(panel)
```

Comment:
Better to get rid of dead code. The codebase is getting p big so these will get annoying to read in the future.